otelcollector/fluent-bit/src/telemetry.go

```diff
@@ -463,6 +471,8 @@ func GetAndSendContainerCPUandMemoryFromCadvisorJSON(container Container, cpuMet
 
 	// Abbreviated properties to save telemetry cost
 	metricTelemetryItem.Properties[memMetricName] = fmt.Sprintf("%d", int(memoryRssBytesLinux))
+	// Updating the pod name from the podRef from Cadvisor Json output since the podname environment variable points to the pod from which the cadvisor endpoint is curl-ed
+	CommonProperties["podname"] = fmt.Sprintf("%s", podName)
```

Comment:
Thanks for fixing this Soham. podname already exists today. I think none of our dashboards use this and today it just storing the value of the pod in which the plugin is running and sending data from(which is ama-metrics-node-* pod). But is it possible to rename it to something else so that it is not confusing? Because at every other place podname is always the pod from which the telemetry is being sent.

Reply:
Thanks Rashmi! Yes I was thinking about this to have a different dimension name instead of updating the existing podName dimension. Let me update this.